frontend/src/components/dashboard/utils/charts.jsx

```diff
@@ -27,12 +27,23 @@ const colors = getRandomColorsArray(10, true);
 
 // bar charts
 
-export const JobStatusBarChart = React.memo(() => {
+export const JobStatusBarChart = React.memo((props) => {
   console.debug("JobStatusBarChart rendered!");
+  /* eslint-disable */
+  var sendURL = JOB_AGG_STATUS_URI;
+  const parameter = props.sendProp;
+  const getValue = parameter.key;
+  console.debug("props is ", getValue);
+  /* eslint-enable */
+
+  /* eslint-disable */
+  sendURL = `${JOB_AGG_STATUS_URI}?org=${getValue}`;
```

Comment:
Pls use a more specific name for this const, what do you think about ORG_JOB_AGG_STATUS_URI ?

Reply:
sure!